src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1495,6 +1487,31 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string) {
 }
 #endif  // EXTERNAL_PROPRIETARY_MODE
 
+void PolicyHandler::OnSnapshotCreatedNextRetry(const BinaryMessage& pt_string) {
```

Comment:
@LitvinenkoIra 
In fact this is copy-paste of HTTP part + part of existing OnSnapshotCreated() method. Instead of adding new interface I suggest to rework existing one. E.g.

```
void PolicyHandler::OnSnapshotCreated(const BinaryMessagePtr& msg) {
#ifdef PROPRIETARY
if (msg) {
   SaveSnapshot;
   SendBCPolicyUpdate;
}
#endif // PROPRIETARY

<common part> 
 ChooseApplication;
 ChooseUrl;
 SendOnSystemRequest
 SetStatus;
}
```

BinaryMessagePtr is shared pointer because of my note below